scripts/build-package.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+const glob = require('glob');
+const rimraf = require('rimraf');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const argv = require('yargs').argv;
+
+/**
+ * Runs the build-babel or build-webpack npm target for the specified package.
+ * @param options
+ * @param options.npmTask Name of the npm task to run
+ * @param options.watch Whether to apply watch argument
+ * @param options.packageName Name of the React package to build
+ */
+function runBuildTask(options) {
+  const task = options.npmTask || 'build-babel';
+  const args = ['run', task];
+  if (options.watch) {
+    args.push('--', '--watch');
+  }
+
+  const child = spawn('npm', args, {
+    cwd: __dirname,
+    env: Object.assign({}, process.env, {
+      PACKAGE: options.packageName,
+    }),
+  });
+
+  child.stdout.on('data', (data) => {
+    process.stdout.write(data);
+  });
+
+  child.stderr.on('data', (data) => {
+    process.stderr.write(data);
+  });
+
+  child.on('close', (code) => {
+    if (code) {
+      process.stderr.write(`${options.packageName} exited with code ${code}`);
+    }
+  });
+}
+
+/**
+ * Build CP only
+ * @param watch Apply watch argument
+ */
+function runBuildPlaygroundTask(watch) {
+  // Build CP
+  runBuildTask({
+    packageName: 'react-component-playground',
+    npmTask: 'build-webpack',
+    watch,
+  });
+}
+
+/**
+ * Run the build for all packages, the CP requires all other components
+ * to be built first.
+ * @param packageNames List of package names
+ */
+function runBuildAllTask(packageNames) {
+  // Cleanup
+  glob.sync('./packages/*/lib').forEach((packageLibPath) => {
+    rimraf.sync(packageLibPath);
+    console.log('WARNING: Removed lib directory for', packageLibPath);
+  });
+
+  // Build all packages except CP
+  packageNames
+      .filter(pkg => pkg !== 'react-component-playground')
+      .forEach(packageName => (
+          runBuildTask({
+            packageName,
+          })
+        ));
+
+  runBuildPlaygroundTask();
```

Comment:
Maybe use [child-process-promise](https://github.com/patrick-steele-idem/child-process-promise), have `runBuildTask` return a promise, and then

```js
Promise.all(
  packageNames
    .filter(pkg => pkg !== 'react-component-playground')
    .map(packageName => runBuildTask({ packageName })
).then(runBuildPlaygroundTask);
```